src/components/tree-item/ch-tree-item.tsx

```diff
@@ -25,7 +27,7 @@ export class ChTreeItem {
   /**
    * Set this attribute if you want the ch-tree-item to display a checkbox
    */
-  @Prop() readonly checkbox: boolean = false;
+  @Prop() checkbox: boolean = false;
```

Comment:
Add `{ mutable: true }` in the Prop decorator

Reply:
Ok. Do you know why I didn't get a warning from Stencil? Just wondering.

Reply:
Do you have the recommended extensions?
https://github.com/genexuslabs/chameleon-controls-library/blob/main/.vscode/extensions.json

Reply:
Yes. I will have to research about this. Thanks.